src/ducks/reimbursements/NoReimbursedExpenses.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { GROUP_DOCTYPE } from 'doctypes'
+import Padded from 'components/Padded'
+import StoreLink from 'components/StoreLink'
+import { KonnectorChip } from 'components/KonnectorChip'
+import { getCategoryName } from 'ducks/categories/categoriesMap'
+import Caption from 'ducks/reimbursements/Caption'
+
+const NoReimbursedExpenses = ({ hasHealthBrands, doc }) => {
+  const { t } = useI18n()
+
+  const categoryName = doc.categoryId ? getCategoryName(doc.categoryId) : null
+
+  let message
+  if (doc._type === GROUP_DOCTYPE) {
+    message = t('Reimbursements.noReimbursed.group')
+  } else if (categoryName) {
+    message = t(`Reimbursements.noReimbursed.${categoryName}`)
+  } else {
+    message = t('Reimbursements.noReimbursed.generic')
+  }
+
+  return (
+    <Padded className="u-pv-0">
+      <Caption>{message}</Caption>
+      {!hasHealthBrands && categoryName === 'healthExpenses' && (
+        <StoreLink type="konnector" category="insurance">
+          <KonnectorChip konnectorType="health" />
+        </StoreLink>
+      )}
+    </Padded>
+  )
+}
+
+export default React.memo(NoReimbursedExpenses)
```

Comment:
does that component re-render ofter? with the same props?

I don't think so, because the props doc changes. 

So no need to memo it.

https://dmitripavlutin.com/use-react-memo-wisely/